python/pyspark/ml/regression.py

```diff
@@ -608,6 +609,141 @@ class GBTRegressionModel(TreeEnsembleModels):
     .. versionadded:: 1.4.0
     """
 
+@inherit_doc
+class AFTSurvivalRegression(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredictionCol,
+                            HasFitIntercept, HasMaxIter, HasTol):
+    """
+    `https://en.wikipedia.org/wiki/Accelerated_failure_time_model`
```

Comment:
It would be nice to put the link in "see also" or embed it in the doc. The first sentence is usually a summary of the class.

(@GayathriMurali  This is an inline comment.)